ocaml/libs/xapi-stdext/lib/xapi-stdext-std/xstringext.ml

```diff
@@ -81,24 +67,28 @@ module String = struct
         in
         concat "" (fold_right aux string [])
 
-  (** Take a predicate and a string, return a list of strings separated by
-      runs of characters where the predicate was true (excluding those characters from the result) *)
   let split_f p str =
```

Comment:
Can we use https://ocaml.org/p/astring/latest/doc/Astring/String/index.html#val-fields here?

Reply:
I'd rather avoid importing Astring if not needed, the function is well tested with unit tests